cloudman/values.yaml

```diff
@@ -29,19 +30,26 @@ helmsman_config:
             - /dashboard
             - /dashboard/*
         enableInsecureLogin: true
-    - name: cloudve/galaxy-cvmfs-csi
+    cvmfs:
+      name: cloudve/galaxy-cvmfs-csi
       namespace: cvmfs
-    - name: cloudve/galaxy
+    galaxy:
+      name: cloudve/galaxy
       namespace: gvl
+      oidc_client:
+        client_id: 'galaxy-auth'
+        client_secret: 'cf84xfps-npm5-hvmb-ntui4grybqrh'
```

Comment:
Just saw this issue: https://github.com/helm/helm/issues/6520
So if that's implemented, we may be able to always auto-generate a client-secret, because no one would want to set this by hand.
For now, what do you think of making these required values?
https://austindewey.com/2018/12/28/helm-tricks-input-validation-with-required-and-fail/

Reply:
I made it so that the client is public if no secret is set, so not sure about making it required. But it'd be soooo amazing if they finally implemented that feature